src/components/KTreeList/KTreeItem.vue

```diff
@@ -70,7 +71,18 @@ const slots = useSlots()
 
 const hasIcon = computed((): boolean => !props.hideIcons || !!slots['item-icon'])
 
-const handleClick = () => {
+const handleClick = (event: any) => {
+  if (event.target) {
+    const ignoredElements = ['a:not(.disabled)', 'button:not([data-tree-item-trigger]):not(disabled)', 'label', 'input:not(disabled)', 'select:not(disabled)']
```

Comment:
`disabled` -> `:disabled`

And I think we can use a selector group so that we don’t have to use a for loop:

```js
const ignoredElements = 'a:not(.disabled), button:not([data-tree-item-trigger]):not(:disabled), label, input:not(:disabled), select:not(:disabled)'

if (event.target.closest(ignoredElements) !== null) {
  return
}
```

Reply:
Sorry, didn't understand what you mean by making it a string in the previous comment but thanks for clarifying. Updated, much cleaner this way!